docs/aws-sdk-for-php-with-minio.md

```diff
@@ -76,7 +76,18 @@ $presignedRequest = $s3->createPresignedRequest($command, '+10 minutes');
 $presignedUrl =  (string)  $presignedRequest->getUri();
 ```
 
-## 6. Set a Bucket Policy
+## 6. Get a plain URL 
+
+```php
+<?php
+$plainUrl = $s3->getObjectUrl('testbucket', 'testkey');
+/** 
+ * Disclaimer: in order to make this work you'll need to make your object/bucket accesible with public permission
+ * This line of code will not provide you a url with 'X-Amz-Algorithm=[...]&X-Amz-Credential=[...]&X-Amz-Date=[...]&X-Amz-Expires=[...]&X-Amz-SignedHeaders=[...]&X-Amz-Signature=[...]'
+ */
```

Comment:
It will be consistent to put the `disclaimer/instructions to make it work` outside of the code block, something like

```
To get a plain URL, you'll need to make your object/bucket accessible with public permission. Also, note that this line of code will not provide you a url with 'X-Amz-Algorithm=[...]&X-Amz-Credential=[...]&X-Amz-Date=[...]&X-Amz-Expires=[...]&X-Amz-SignedHeaders=[...]&X-Amz-Signature=[...]

<?php
 +$plainUrl = $s3->getObjectUrl('testbucket', 'testkey');

```